src/components/BigButton/BigButton.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import './BigButton.style.scss';
+
+interface BigButtonProps {
+  children: React.ReactNode|React.ReactNodeArray;
+}
+
+function BigButton(props: BigButtonProps) {
```

Comment:
I prefer to use arrow functions in all components

Reply:
Fixed